src/index.js

```diff
@@ -27,6 +27,13 @@ function syncEvent(node, eventName, newEventHandler) {
 }
 
 export default function (CustomElement, opts) {
+  function whitelistAttrs(props) {
```

Comment:
I'm unsure if this is the best name for it, but I don't want to 🚲 shed too much on it.